java-grafeas/.repo-metadata.json

```diff
@@ -1,16 +1,17 @@
 {
-    "api_shortname": "containeranalysis",
-    "name_pretty": "Grafeas",
-    "product_documentation": "https://grafeas.io",
-    "client_documentation": "https://cloud.google.com/java/docs/reference/grafeas/latest/overview",
-    "issue_tracker": "",
-    "api_description": "n/a",
-    "release_level": "stable",
-    "language": "java",
-    "repo": "googleapis/google-cloud-java",
-    "repo_short": "java-grafeas",
-    "distribution_name": "io.grafeas:grafeas",
-    "api_id": "containeranalysis.googleapis.com",
-    "codeowner_team": "@googleapis/aap-dpes",
-    "library_type": "GAPIC_AUTO"
-}
+  "api_shortname": "containeranalysis",
+  "name_pretty": "Grafeas",
+  "product_documentation": "https://grafeas.io",
+  "api_description": "n/a",
+  "client_documentation": "https://cloud.google.com/java/docs/reference/grafeas/latest/overview",
+  "release_level": "stable",
+  "transport": "grpc",
```

Comment:
I see that `transport` and `requires_billing` are new, is it because they are always generated now?

Reply:
Yes.

Reply:
Where do we get the default value `false` for `requires_billing`? I thought it is [true](https://github.com/googleapis/sdk-platform-java/blob/38b90e296505fdca413747f57741c4278f6e3738/library_generation/model/generation_config.py#L138)?

Reply:
The default value is `true`.

I set it to `false` in generation config.

Reply:
I think we should keep it as `true` because it does requires billing if you go to the [api console](https://console.cloud.google.com/flows/enableapi?apiid=containeranalysis.googleapis.com). Or we can leave it as not generated for now, and change the generator logic to not generate `requires_billing` if it is not specified.

Reply:
If it requires billing, I'll change it to `true` by removing it from config. The value is `true` by default.

Reply:
Done.